View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
The right solution IMO would be to have a method ``getSymfonySerializationContext`` or something like that, returning the context array expected by Symfony (and it could be returned for any version btw, as Symfony will ignore unknown keys and this would allow people to provide other context options too)